logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/UserSessionScope.kt

```diff
@@ -970,6 +972,12 @@ class UserSessionScope internal constructor(
             persistMessage = persistMessage
         )
 
+    private val conversationMessageTimerEventHandler: ConversationMessageTimerEventHandler
```

Comment:
I think messageTimer does not explain the purpose of this well enough, for someone outside the context